[W12-2] Debt Tracker

aaditkamat opened the pull request:
This PR contains all the changes we added to the project documentation and different components of the code base. @xiaoyeong @Bellaaarh @weiqing-nic @julianc269

damithc commented:
You have extra spaces in the team ID :-)

damithc commented:
You also haven't _mentioned_ the team members in the PR description.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -14,247 +14,80 @@ ifdef::env-github[]
-Returns `john` and `John Doe`
-* `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
+Finds persons whose names contain any of the given keywords.
```
Might be good to give examples of usage as in AB4

asiddharth commented on README.adoc:
```diff
@@ -15,25 +8,32 @@ ifndef::env-github[]
 == Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
+* This app was based off https://github.com/nus-cs2103-AY1819S1/addressbook-level4 [Address Book Level 4].
```
Good job acknowledging AB4!

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -103,6 +103,147 @@ When you are ready to start coding,
+
+|===
+
 == Design
```
Might be good to add use cases, NFRs to your DG

asiddharth reviewed (commented):
Hi team,
You shave a lot of changes relating to GUI (mostly documentation I hope). Please note that the input should be via command line (or should have a CLI alternative, if it is a GUI). It is recommended to not put in much effort into designing a GUI input as these will not be graded. 

I have also included some comments for the Dev guide, User guide.
Good luck!

asiddharth commented on src/main/java/seedu/address/logic/commands/SelectCommand.java:
```diff
@@ -13,12 +13,16 @@
 import seedu.address.model.Model;
 import seedu.address.model.person.Person;
 
+
```
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

asiddharth commented on src/main/java/seedu/address/logic/commands/RedoCommand.java:
```diff
@@ -7,12 +7,15 @@
 import seedu.address.model.Model;
 
+
+
```
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

asiddharth commented on src/main/java/seedu/address/logic/commands/HistoryCommand.java:
```diff
@@ -8,12 +8,15 @@
 import seedu.address.logic.CommandHistory;
 import seedu.address.model.Model;
 
+
```
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

asiddharth reviewed (commented):
Minor comments. Please try to add tests for your changes as well

asiddharth commented on README.adoc:
```diff
@@ -1,11 +1,4 @@
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
-https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
```
You may want to add back these links updated to link to the travis/appvoyer page for your product/application

asiddharth commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,66 @@
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Cheong Kit Wai, Julian
+image::julian.png[width="150", align="left"]
+{empty}[https://github.com/julianc269[github]]
```
May be helpful to add the portfolio for each team member. Will be useful to keep everything updated as you progress with the project.

https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/project-deliverables.html#deliverable-project-portfolio-page-ppp

asiddharth commented on docs/ContactUs.adoc:
```diff
@@ -2,6 +2,4 @@
-* *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level4/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
-* *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at `shawyeong94 [at] gmail.com'
```
Good job here!

asiddharth commented on README.adoc:
```diff
@@ -15,25 +8,32 @@ ifndef::env-github[]
 
 == Site Map
 
 * <<UserGuide#, User Guide>>
```
Links broken. Please update

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -14,247 +14,160 @@ ifdef::env-github[]
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Qui ...
+Debt Tracker (DT) is for those *who prefer to use a desktop app for tracking the user’s loans.*
```
Nice intro!

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -14,247 +14,160 @@ ifdef::env-github[]
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
+1. Ensure you have Java version 9 or later installed in your Computer.
+2. Download the latest debttracker.jar.
```
Can include a link to the release (like in AB4).

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -14,247 +14,160 @@ ifdef::env-github[]
-== Command Summary
+Redo : redo
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
```
This section is useful for any new user. May be good to add it back.

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -14,247 +14,160 @@ ifdef::env-github[]
+Undo : undo
 
-== Command Summary
+Redo : redo
```
Any new features that you plan to add? 

Any "coming in v2.0" features?

Please note that before your "test demo", it is recommended to update the UG and DG with all the features so that the user can test it out.

asiddharth reviewed (commented):
There is a lot of work to be done!! Here are a few comments that may be helpful.

1. Add a description to the product.
2. In the UG, there may be some formatting issues
![12-2](https://user-images.githubusercontent.com/23400124/46578299-76310d80-ca2f-11e8-989e-7ff896b9fc04.png)

Possible to separate the command format from its description ? Could also help to have some examples as before.

3. The following items seem missing in the DG :
Value Proposition
Project scope
Use cases
NFRs

asiddharth reviewed (commented):
Hi Team, Here are some suggestions for this week.
1. 
![w12-2](https://user-images.githubusercontent.com/23400124/47269161-b7c3cb80-d58c-11e8-9fbb-29bae3c6051a.png)
As your project is a morph of AB4, it would be better to remove the links to AB1, AB2, AB3 and AB4 in your autopublished website.

2. 
![w12-2-pr](https://user-images.githubusercontent.com/23400124/47269198-0ec9a080-d58d-11e8-8f57-79e76147b83f.png)
(a) PLease fix your build.
(b) Everyone please fix your photos. Please upload a .png file . You can verify if it shows at https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/projectList.html

Some suggestions regarding DG :

Your DG looks very similar to that of the parent repository. You might want to consider doing some of the following things.

1. You might want to add some architecure, sequence and class diagrams tailored to your project application.
2. You might want to add some feature implementation details.
3. Many references to AB4. Could help to remove references related to AB4 not relevant to your project application.
4. Add NFRs
5. Use cases missing

Recommended to finish all the above-mentioned things soon
A lot of work to do before the testing session. Good luck!

weiqing-nic committed:
Fix Bugs and User able to delete

weiqing-nic committed:
Merge pull request #76 from weiqing-nic/new

Able to delete and fix bugs

weiqing-nic committed:
Styling Fix

weiqing-nic committed:
Merge pull request #77 from weiqing-nic/new

Styling Fix

xiaoyeong committed:
add new NextTransactionCommand and it's test case

xiaoyeong committed:
fixed checkstyle error

xiaoyeong committed:
fix comments and message usage

Commit:
Add sort command

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Add tests for sort command and modify assertCommandSuccess/assertCommandFailure
test methods

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Fixed checkstyle bug in main module classes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Merge branch 'master' of github.com:CS2103-AY1819S1-W12-2/main into aadit_changes

Commit:
Fix checkstyle bugs in test module classes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

julianc269 committed:
Merge pull request #80 from CS2103-AY1819S1-W12-2/aadit_changes

Add sort command and tests

julianc269 committed:
Merge pull request #81 from CS2103-AY1819S1-W12-2/master

merge master into julian_changes

julianc269 committed:
Update FilterCommand

julianc269 committed:
Add Tests for Filter Command

julianc269 committed:
CalendarCommand: add sync command

xiaoyeong committed:
Merge branch 'master' into shaw_yeong_changes

julianc269 committed:
Update User Guide

julianc269 committed:
Update FilterCommand MESSAGE_USAGE

xiaoyeong committed:
fix test case

julianc269 committed:
CalendarManager: sync calendar upon login

xiaoyeong committed:
update userguide

xiaoyeong committed:
fix error in userguide

Commit:
Merge pull request #79 from CS2103-AY1819S1-W12-2/shaw_yeong_changes

add NextTransactionCommand and test cases

julianc269 committed:
Update FilterCommand MESSAGE_USAGE

julianc269 committed:
Merge branch 'master' into julian_changes

julianc269 committed:
convertCurrency optimization fix

xiaoyeong committed:
update develop guide

Bellaaarh committed:
Trying to fix Bella's test fails

Bellaaarh committed:
Implement wildcard search

Bellaaarh committed:
Update UG

Commit:
Merge pull request #88 from Bellaaarh/trying_to_fix

Update UG/DG and WildCardSearch command

julianc269 committed:
Merge branch 'master' into julian_changes

Commit:
Merge pull request #82 from CS2103-AY1819S1-W12-2/julian_changes

Update FilterCommand and CalendarCommand

Commit:
Add an AnalyticsCommand test to FinancialDatabaseParserTest class

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Bellaaarh committed:
Deleted Find Command and updated UG accordingly. Changed showTransactionsWithName (in FinancialDatabaseSystemTest) to use wildcard search instead of find command.

Bellaaarh committed:
Merge branch 'master' into trying_to_fix

Bellaaarh committed:
fix checkstyle error

Bellaaarh committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into trying_to_fix

# Conflicts:
#	docs/UserGuide.adoc

Bellaaarh committed:
Merge branch 'trying_to_fix' of https://github.com/Bellaaarh/addressbook-level4 into trying_to_fix

# Conflicts:
#	docs/UserGuide.adoc

Commit:
Update UserGuide.adoc

Commit:
Add more whitespaces in Command Summary section

This enhances the readability of the examples from the point of view of the user.

Commit:
Fix capitalization

Commit:
Merge pull request #89 from Bellaaarh/trying_to_fix

Delete Find Command

Commit:
Merge branch 'master' of github.com:CS2103-AY1819S1-W12-2/main into aadit_changes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

# Conflicts:
#	src/test/java/seedu/address/logic/parser/FinancialDatabaseParserTest.java

Commit:
Merge branch 'master' of github.com:CS2103-AY1819S1-W12-2/main into aadit_changes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

# Conflicts:
#	src/test/java/seedu/address/logic/parser/FinancialDatabaseParserTest.java

Commit:
Remove trailing whitespaces in the User Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Add more tests in FinancialDatabaseParserTest class

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Fix checkstyle bugs in User Guide and add more tests (#91)

* Add an AnalyticsCommand test to FinancialDatabaseParserTest class

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Merge branch 'master' of github.com:CS2103-AY1819S1-W12-2/main into aadit_changes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

# Conflicts:
#	src/test/java/seedu/address/logic/parser/FinancialDatabaseParserTest.java

* Remove trailing whitespaces in the User Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Add more tests in FinancialDatabaseParserTest class

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Fixed typo in one of the links in the Acknowledgements section

Commit:
Update responsibilities of team members in AboutUs.adoc

xiaoyeong committed:
Merge pull request #95 from CS2103-AY1819S1-W12-2/master

merge from master

Bellaaarh committed:
Refactored

Commit:
Fix incorrect change in the License file

weiqing-nic committed:
Merge pull request #13 from CS2103-AY1819S1-W12-2/master

merge

weiqing-nic committed:
Fix style changes

Bellaaarh committed:
Made all commands undoable and changed tests to reflect the undoable commands. Also refactored several methods in FinancialDatabase (previous PR). Also added two commands, PaidCommand and PastCommand (incomplete)

Commit:
Merge pull request #84 from CS2103-AY1819S1-W12-2/wei_qing_changes

Update Photo Class

Bellaaarh committed:
Halfway implementing GUI tab for past transactions.

Bellaaarh committed:
Merge remote-tracking branch 'CS2103-AY-1819S1-W12-2/master' into trying_to_fix

Bellaaarh committed:
Update UG and DEFAULT_PHOTO in Photo.java

Bellaaarh committed:
Fixed checkstyle errors

Bellaaarh committed:
Updated UG

Commit:
Merge pull request #96 from Bellaaarh/trying_to_fix

Add new undoable commands

Commit:
Merge master into branch aadit_changes

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Updated conditions for the relevant commands in the User Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Change regex and format of value after calculation of interest.

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Add information about external API within the User Guide.

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

weiqing-nic committed:
Changes to photocommand

weiqing-nic committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new

weiqing-nic committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new

weiqing-nic committed:
Merge branch 'new' of https://github.com/weiqing-nic/debt-tracker into new

weiqing-nic committed:
Changes

Commit:
Make changes in commands to only allow conversion through convert
command

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Rename photo path so that the default photo is not shown for user testing

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Fix checkstyle for Amount class

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

codacy-badger committed:
Add Codacy badge

Commit:
Merge pull request #97 from CS2103-AY1819S1-W12-2/aadit_changes

Update commands and add some more changes for the User Guide

Commit:
Replace the screenshot of UI in Readme

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Merge pull request #100 from codacy-badger/codacy-badge

Add a Codacy badge to README.adoc

Commit:
Shift the Codacy Badge to required position

Bellaaarh committed:
Merge pull request #101 from CS2103-AY1819S1-W12-2/aadit_changes

Replace the screenshot of UI in Readme

Commit:
Add portfolio for publishing

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Remove trailing whitespaces

Signed-off-by: Aadit <aadit.k12@gmail.com>

Bellaaarh committed:
Update GUI. Should have a list under Past Transactions tab. Dimensions still not correct though.

Bellaaarh committed:
Update GUI under Past Transactions tab.

Bellaaarh committed:
Merge remote-tracking branch 'CS2103-AY-1819S1-W12-2/aadit_changes' into trying_to_fix

Bellaaarh committed:
fixed white trailing space.

Bellaaarh committed:
Fixing checkstyle errors.

Bellaaarh committed:
Merge pull request #102 from CS2103-AY1819S1-W12-2/aadit_changes

Add portfolio for publishing

Bellaaarh committed:
Merge remote-tracking branch 'CS2103-AY-1819S1-W12-2/master' into trying_to_fix

Bellaaarh committed:
fixing checkstyle errors

xiaoyeong committed:
Merge pull request #119 from CS2103-AY1819S1-W12-2/master

merge from master

xiaoyeong committed:
Add portfolio for publishing

Commit:
Add cap to interest rate and update user guide.

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Remove alias for InterestCommand

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Display portfolio in AboutUs.adoc

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Add tests for Interest class and modify other tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Modify existing tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Make changes suggested by Codacy

Signed-off-by: Aadit <aadit.k12@gmail.com>

Bellaaarh committed:
Able to store past transactions list now. Paid Command now works and can be undone properly. Left with Past Command. Also, previous errors should be fixed now.

Bellaaarh committed:
Fixing checkstyle errors

Bellaaarh committed:
Fix checkstyle errors

Bellaaarh committed:
Merge pull request #118 from Bellaaarh/trying_to_fix

Changed GUI for PastTransactions

Bellaaarh committed:
Update Photo.java default photo file path. Abstracted resetData.

Bellaaarh committed:
Merge pull request #120 from CS2103-AY1819S1-W12-2/aadit_changes

Add tests and update user guide

Bellaaarh committed:
Fix checkstyle error

julianc269 committed:
Merge pull request #122 from CS2103-AY1819S1-W12-2/master

Merge master into julian_changes

weiqing-nic committed:
Able to undo and redo

julianc269 committed:
FilterCommand: Filter Past Transactions

weiqing-nic committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new

julianc269 committed:
FilterCommand: Add filter by tag

weiqing-nic committed:
Checkstyle

julianc269 committed:
Calendar synchronization for clear, undo and redo commands

julianc269 committed:
Modify CalendarManager and CalendarCommand

julianc269 committed:
Fix Checkstyle errors

weiqing-nic committed:
Checkstyle

julianc269 committed:
Add CalendarCommandParser Test

julianc269 committed:
Fix Checkstyle errors

weiqing-nic committed:
Gradle cannot create folder

julianc269 committed:
Add FilterCommand test case

weiqing-nic committed:
coverage

weiqing-nic committed:
coverage

Commit:
Fix checkstyle in Developer Guide

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Remove command aliases

Signed-off-by: Aadit <aadit.k12@gmail.com>

Commit:
Rename person to transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

weiqing-nic committed:
coverage

weiqing-nic committed:
try travis

weiqing-nic committed:
travis

Commit:
Add tests for FinancialDatabaseParser

Signed-off-by: Aadit <aadit.k12@gmail.com>

Bellaaarh committed:
Remove ToDoList-related files. Now Delete, Select and WildcardSearch commands are able to be used for past transactions list. UG has been updated to reflect this. Halfway working on PaidCommand Test.

weiqing-nic committed:
TestCase

weiqing-nic committed:
add Picture

Bellaaarh committed:
Merge pull request #125 from weiqing-nic/new

Photo able to undo and redo

weiqing-nic committed:
Merge pull request #124 from CS2103-AY1819S1-W12-2/julian_changes

Modify filter and calendar command

Commit:
Resolve merge conflicts with CS2103-AY1819S1-W12-2/master

weiqing-nic committed:
Generic Cases

Bellaaarh committed:
Changed PastCommand to SwitchCommand so users can switch between Past Transactions tab and Current Transactions tab. Halfway writing PaidCommandTest. Updated UserGuide to reflect changes in PastCommand (now SwitchCommand).

Bellaaarh committed:
Merge pull request #127 from weiqing-nic/new

Generic Cases

Bellaaarh committed:
Merge pull request #126 from CS2103-AY1819S1-W12-2/aadit_changes

Add more tests for better coverage

Bellaaarh committed:
Merge remote-tracking branch 'CS2103-AY-1819S1-W12-2/master' into updated

# Conflicts:
#	src/main/java/seedu/address/logic/commands/PastCommand.java
#	src/main/java/seedu/address/logic/parser/FinancialDatabaseParser.java
#	src/main/java/seedu/address/model/person/Photo.java
#	src/main/java/seedu/address/ui/PastTransactionListPanel.java

Bellaaarh committed:
Made FilterCommand undoable. Made SelectCommand for Past Transaction List undoable. SwitchCommand is no longer undoable.

Bellaaarh committed:
Add some tests under PaidCommandTest.java for PaidCommand

Bellaaarh committed:
Fix checkstyle errors

Bellaaarh committed:
Fix checkstyle error

Bellaaarh committed:
Fix checkstyle error

Bellaaarh committed:
Fix checkstyle errors.

Bellaaarh committed:
Fix checkstyle error

weiqing-nic committed:
Merge pull request #121 from Bellaaarh/updated

Update Photo.java default photo file path. Abstracted resetData.

weiqing-nic committed:
Merge pull request #16 from CS2103-AY1819S1-W12-2/master

merge

weiqing-nic committed:
changes

asiddharth reviewed (commented):
Some comments to improve documentation.
1. These links dont seem to work.
![r_update_links](https://user-images.githubusercontent.com/23400124/48133458-c53dbd00-e2d1-11e8-8fdc-14e4af2c639d.png)

2. Good class diagram for the Model component. Please consider a different color for the Model, ModelManager and UserPref entities to make it more legible. Also please consider enlarging the image a bit.
![diag](https://user-images.githubusercontent.com/23400124/48134307-8826fa00-e2d4-11e8-9329-0a65f633af44.png)
Usage of aggregations is not encouraged, but it is ok. Also, you may want to combine the aggregations ModelManager->VersionedFinancialDatabase and ModelManager->UserPref into one aggregation

3. This diagram contains info about the undo/redo of AB4. You may want to rectify it by using VersionedFinancialDatabase etc.
![vab](https://user-images.githubusercontent.com/23400124/48134826-1bacfa80-e2d6-11e8-9776-c4367f730611.png)

4. Same here
![vab](https://user-images.githubusercontent.com/23400124/48134860-354e4200-e2d6-11e8-9c9a-bc56b6afee5d.png)

5. You might want to add details of other features under the design considerations section. Also, using sequence diagrams will be useful.

6. Use cases missing in DG
     Add use cases you had before. Also consider adding activity diagrams where relevant.

7. Would be good to add more test cases. If not remove "{ more test cases …​ }"
![mtc](https://user-images.githubusercontent.com/23400124/48135119-f076db00-e2d6-11e8-9df9-44ea0461182c.png)

xiaoyeong committed:
add PPP for xiaoyeong

weiqing-nic committed:
change

xiaoyeong committed:
Merge pull request #130 from weiqing-nic/new

New

xiaoyeong committed:
update developer guide for use cases

Commit:
Fix the links in the SiteMap section

Commit:
Remove more test cases comment

xiaoyeong committed:
improve PPP

xiaoyeong committed:
Merge pull request #131 from CS2103-AY1819S1-W12-2/shaw_yeong_changes

add PPP for xiaoyeong

weiqing-nic committed:
Merge pull request #17 from CS2103-AY1819S1-W12-2/master

merge lastest

Commit:
Fix PE dry run issues and add tests (#132)

* Add checks for Interest and Convert commands

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add a check for the Delete Command

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add new message for duplicate transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add test for deadline

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for transactions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix amount bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix convert command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix interest command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Update condition for interest command in UG

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Edit messages

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove unused import

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix code according to Codacy suggestions

Signed-off-by: Aadit <aadit.k12@gmail.com>

julianc269 committed:
Merge pull request #133 from CS2103-AY1819S1-W12-2/master

merge from master

julianc269 committed:
ModelManager: Sync Calendar on commit/undo/redo

julianc269 committed:
Add more filter test cases

julianc269 committed:
Update filter usage message

julianc269 committed:
Update UserGuide for Calendar and Filter Commands

julianc269 committed:
Update FilterCommand parser

julianc269 committed:
Add FilterCommandParser test cases

julianc269 committed:
Fix checkstyle errors

julianc269 committed:
Update User and Developer Guide

julianc269 committed:
UserGuide: Filter does not work with undo/redo

julianc269 committed:
FilterCommand: remove commiting to database

julianc269 committed:
CalendarManager: remove unused code

julianc269 committed:
Fix error

julianc269 committed:
Fix checkstyle error

julianc269 committed:
Fix code style

julianc269 committed:
Remove unnecessary photos

julianc269 committed:
Update gitignore

julianc269 committed:
Fix Checkstyle Error

weiqing-nic committed:
Merge pull request #134 from CS2103-AY1819S1-W12-2/julian_changes

Update FilterCommand, UGDG, and Add Test Cases

weiqing-nic committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new
This reverts commit 37b9bd81f6ec92a4da5767f2045ccfc1f0bc18a2.

weiqing-nic committed:
Merge pull request #18 from CS2103-AY1819S1-W12-2/master

merge

weiqing-nic committed:
Merge branch 'new' of https://github.com/weiqing-nic/debt-tracker into new

julianc269 committed:
Calendar: clear browser panel when logged out

julianc269 committed:
Fix checkstyle errors

weiqing-nic committed:
PPP merge

Bellaaarh committed:
Add PPP (#135)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

xiaoyeong committed:
update PPP for xiaoyeong

xiaoyeong committed:
update PPP

xiaoyeong committed:
update PPP

xiaoyeong committed:
update PPP

weiqing-nic committed:
PPP merg3

weiqing-nic committed:
new pull

Bellaaarh committed:
Merge pull request #137 from CS2103-AY1819S1-W12-2/shaw_yeong_changes

update PPP for xiaoyeong

Bellaaarh committed:
Update PPP  (#136)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

weiqing-nic committed:
changes

weiqing-nic committed:
New (#138)

* PPP merge

* PPP merg3

* new pull

* changes

Bellaaarh committed:
Deleted ysabellatan.adoc

weiqing-nic committed:
Link error

Bellaaarh committed:
Update AboutUs.adoc

weiqing-nic committed:
Merge branch 'master' into new

Bellaaarh committed:
Update UG and bellaaarh.adoc

Bellaaarh committed:
Merge pull request #139 from weiqing-nic/new

New

xiaoyeong committed:
update about us

Bellaaarh committed:
Update AboutUs.adoc (#140)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

Bellaaarh committed:
Update bellaaarh.adoc

xiaoyeong committed:
merge from master (#142)

* Fix style changes

* Changes to photocommand

* Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new

* Changes

* Update GUI. Should have a list under Past Transactions tab. Dimensions still not correct though.

* Update GUI under Past Transactions tab.

* fixed white trailing space.

* Fixing checkstyle errors.

* fixing checkstyle errors

* Add cap to interest rate and update user guide.

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove alias for InterestCommand

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Display portfolio in AboutUs.adoc

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for Interest class and modify other tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Modify existing tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Make changes suggested by Codacy

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Able to store past transactions list now. Paid Command now works and can be undone properly. Left with Past Command. Also, previous errors should be fixed now.

* Fixing checkstyle errors

* Fix checkstyle errors

* Update Photo.java default photo file path. Abstracted resetData.

* Fix checkstyle error

* Able to undo and redo

* FilterCommand: Filter Past Transactions

* FilterCommand: Add filter by tag

* Checkstyle

* Calendar synchronization for clear, undo and redo commands

* Modify CalendarManager and CalendarCommand

* Fix Checkstyle errors

* Checkstyle

* Add CalendarCommandParser Test

* Fix Checkstyle errors

* Gradle cannot create folder

* Add FilterCommand test case

* coverage

* coverage

* Fix checkstyle in Developer Guide

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove command aliases

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Rename person to transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* coverage

* try travis

* travis

* Add tests for FinancialDatabaseParser

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove ToDoList-related files. Now Delete, Select and WildcardSearch commands are able to be used for past transactions list. UG has been updated to reflect this. Halfway working on PaidCommand Test.

* TestCase

* add Picture

* Generic Cases

* Changed PastCommand to SwitchCommand so users can switch between Past Transactions tab and Current Transactions tab. Halfway writing PaidCommandTest. Updated UserGuide to reflect changes in PastCommand (now SwitchCommand).

* Made FilterCommand undoable. Made SelectCommand for Past Transaction List undoable. SwitchCommand is no longer undoable.

* Add some tests under PaidCommandTest.java for PaidCommand

* Fix checkstyle errors

* Fix checkstyle error

* Fix checkstyle error

* Fix checkstyle errors.

* Fix checkstyle error

* changes

* change

* Fix the links in the SiteMap section

* Remove more test cases comment

* Fix PE dry run issues and add tests (#132)

* Add checks for Interest and Convert commands

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add a check for the Delete Command

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add new message for duplicate transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add test for deadline

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for transactions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix amount bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix convert command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix interest command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Update condition for interest command in UG

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Edit messages

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove unused import

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix code according to Codacy suggestions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* ModelManager: Sync Calendar on commit/undo/redo

* Add more filter test cases

* Update filter usage message

* Update UserGuide for Calendar and Filter Commands

* Update FilterCommand parser

* Add FilterCommandParser test cases

* Fix checkstyle errors

* Update User and Developer Guide

* UserGuide: Filter does not work with undo/redo

* FilterCommand: remove commiting to database

* CalendarManager: remove unused code

* Fix error

* Fix checkstyle error

* Fix code style

* Remove unnecessary photos

* Update gitignore

* Fix Checkstyle Error

* PPP merge

* Add PPP (#135)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* PPP merg3

* new pull

* Update PPP  (#136)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

* changes

* New (#138)

* PPP merge

* PPP merg3

* new pull

* changes

* Link error

* Update AboutUs.adoc (#140)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

xiaoyeong committed:
update PPP and userguide

weiqing-nic committed:
ppp error

weiqing-nic committed:
Merge branch 'new' of https://github.com/weiqing-nic/debt-tracker into new

xiaoyeong committed:
udate userguide

Bellaaarh committed:
Update PPP and UG

xiaoyeong committed:
Merge branch 'master' into shaw_yeong_changes

Bellaaarh committed:
Update UG. (#141)

* Deleted ysabellatan.adoc

* Update AboutUs.adoc

* Update UG and bellaaarh.adoc

* Update bellaaarh.adoc

* Update PPP and UG

xiaoyeong committed:
update about us (#143)

* update about us

* merge from master (#142)

* Fix style changes

* Changes to photocommand

* Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-2/main into new

* Changes

* Update GUI. Should have a list under Past Transactions tab. Dimensions still not correct though.

* Update GUI under Past Transactions tab.

* fixed white trailing space.

* Fixing checkstyle errors.

* fixing checkstyle errors

* Add cap to interest rate and update user guide.

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove alias for InterestCommand

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Display portfolio in AboutUs.adoc

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for Interest class and modify other tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Modify existing tests

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Make changes suggested by Codacy

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Able to store past transactions list now. Paid Command now works and can be undone properly. Left with Past Command. Also, previous errors should be fixed now.

* Fixing checkstyle errors

* Fix checkstyle errors

* Update Photo.java default photo file path. Abstracted resetData.

* Fix checkstyle error

* Able to undo and redo

* FilterCommand: Filter Past Transactions

* FilterCommand: Add filter by tag

* Checkstyle

* Calendar synchronization for clear, undo and redo commands

* Modify CalendarManager and CalendarCommand

* Fix Checkstyle errors

* Checkstyle

* Add CalendarCommandParser Test

* Fix Checkstyle errors

* Gradle cannot create folder

* Add FilterCommand test case

* coverage

* coverage

* Fix checkstyle in Developer Guide

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove command aliases

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Rename person to transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* coverage

* try travis

* travis

* Add tests for FinancialDatabaseParser

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove ToDoList-related files. Now Delete, Select and WildcardSearch commands are able to be used for past transactions list. UG has been updated to reflect this. Halfway working on PaidCommand Test.

* TestCase

* add Picture

* Generic Cases

* Changed PastCommand to SwitchCommand so users can switch between Past Transactions tab and Current Transactions tab. Halfway writing PaidCommandTest. Updated UserGuide to reflect changes in PastCommand (now SwitchCommand).

* Made FilterCommand undoable. Made SelectCommand for Past Transaction List undoable. SwitchCommand is no longer undoable.

* Add some tests under PaidCommandTest.java for PaidCommand

* Fix checkstyle errors

* Fix checkstyle error

* Fix checkstyle error

* Fix checkstyle errors.

* Fix checkstyle error

* changes

* change

* Fix the links in the SiteMap section

* Remove more test cases comment

* Fix PE dry run issues and add tests (#132)

* Add checks for Interest and Convert commands

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add a check for the Delete Command

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add new message for duplicate transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add test for deadline

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for transactions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix amount bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix convert command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix interest command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Update condition for interest command in UG

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Edit messages

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove unused import

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix code according to Codacy suggestions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* ModelManager: Sync Calendar on commit/undo/redo

* Add more filter test cases

* Update filter usage message

* Update UserGuide for Calendar and Filter Commands

* Update FilterCommand parser

* Add FilterCommandParser test cases

* Fix checkstyle errors

* Update User and Developer Guide

* UserGuide: Filter does not work with undo/redo

* FilterCommand: remove commiting to database

* CalendarManager: remove unused code

* Fix error

* Fix checkstyle error

* Fix code style

* Remove unnecessary photos

* Update gitignore

* Fix Checkstyle Error

* PPP merge

* Add PPP (#135)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* PPP merg3

* new pull

* Update PPP  (#136)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

* changes

* New (#138)

* PPP merge

* PPP merg3

* new pull

* changes

* Link error

* Update AboutUs.adoc (#140)

* Update DeveloperGuide. Update Javadoc for PaidCommand, SwitchCommand, SwitchTabsEvent, PastTransactionBrowserPanel, PastTransactionListPanel. Removed an unnecessary comment from MainWindow.java. Add Ysabella's PPP.

* Update UI picture in README.adoc

* Fix checkstyle error

* Refactored name of PPP file. Added link to Reposense dashboard.

* Add link to PR where DG was updated by me.

* Add new lines to PPP

* Fixed DeveloperGuide format. Updated my portfolio under AboutUs page.

* Removed some issues from PPP

* update PPP and userguide

* udate userguide

Bellaaarh committed:
Merge pull request #144 from weiqing-nic/new

PPP Bug Fix

Commit:
Remove css selector to get rid of trailing URLs after the main hyperlinks

Following the suggestion posted for https://github.com/nus-cs2103-AY1819S1/forum/issues/172

julianc269 committed:
Add PPP

julianc269 committed:
Edit PPP

julianc269 committed:
Fix test

julianc269 committed:
codacy error

julianc269 committed:
fix test

julianc269 committed:
Add PPP (#145)

* Calendar: clear browser panel when logged out

* Fix checkstyle errors

* Add PPP

* Edit PPP

* Fix test

* codacy error

* fix test

Bellaaarh committed:
Update DG and Bella's PPP

weiqing-nic committed:
Merge pull request #146 from Bellaaarh/update4

Update DG and PPP phrasing

Bellaaarh committed:
Add one word to UG under delete command. (#148)

xiaoyeong committed:
update photos

xiaoyeong committed:
Merge branch 'shaw_yeong_changes' of https://github.com/CS2103-AY1819S1-W12-2/main into shaw_yeong_changes

Bellaaarh committed:
Merge pull request #149 from CS2103-AY1819S1-W12-2/shaw_yeong_changes

updata photos

julianc269 committed:
update comments

julianc269 committed:
Merge branch 'master' into julian_changes

julianc269 committed:
Update Julian PPP

julianc269 committed:
Merge branch 'julian_changes' of https://github.com/CS2103-AY1819S1-W12-2/main into julian_changes

julianc269 committed:
Modify comments (#151)

* Calendar: clear browser panel when logged out

* Fix checkstyle errors

* Add PPP

* Edit PPP

* Fix test

* codacy error

* fix test

* update comments

* Update Julian PPP

julianc269 committed:
Fix UserGuide/PPP error

julianc269 committed:
Merge branch 'master' into julian_changes

julianc269 committed:
Fix UserGuide error

julianc269 committed:
Fix Asciidoc stylesheet

julianc269 committed:
revert stylesheet change

Commit:
Update Developer Guide with more use cases and manual user testing instructions (#150)

* Add checks for Interest and Convert commands

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add a check for the Delete Command

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add new message for duplicate transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add test for deadline

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for transactions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix amount bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix convert command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix interest command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Update condition for interest command in UG

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Edit messages

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove unused import

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix code according to Codacy suggestions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add more manual testing instructions to Developer Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Add more use cases to Developer Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Update AmountTest to prevent build fail on different days

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Update error message and refactor constant

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Add a new typical transaction

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Make changes to PPP (#153)

* Add checks for Interest and Convert commands

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add a check for the Delete Command

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add new message for duplicate transaction

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add test for deadline

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add tests for transactions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix amount bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix convert command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix interest command bugs

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Update condition for interest command in UG

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Edit messages

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Remove unused import

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Fix code according to Codacy suggestions

Signed-off-by: Aadit <aadit.k12@gmail.com>

* Add more manual testing instructions to Developer Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Add more use cases to Developer Guide

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Update AmountTest to prevent build fail on different days

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Update error message and refactor constant

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Add a new typical transaction

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Make changes to PPP

julianc269 committed:
Merge pull request #154 from CS2103-AY1819S1-W12-2/master

Merge from master

julianc269 committed:
Edit asciidoctor.css for PPP

weiqing-nic committed:
Merge pull request #152 from CS2103-AY1819S1-W12-2/julian_changes

Fix UserGuide error

julianc269 committed:
Edit PPP (#155)

* Edit PPP

* Fix PPP format issue

* Edit PPP

Commit:
Update aaditkamat.adoc

Add code sections to my portfolio

Commit:
Fix minor styling issue in aaditkamat.adoc

Commit:
Update transaction dates to future dates (#157)

* Update transaction dates to future dates

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Match transaction deadlines to fix tests

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

* Update README.adoc

Signed-off-by: aaditkamat <aadit.k12@gmail.com>

Commit:
Update link in README to render the app UI

commit
